frontend/public/components/nav.jsx

```diff
@@ -374,7 +374,8 @@ export const Navigation = ({ isNavOpen, onNavSelect }) => {
 
         <MonitoringNavSection />
 
-        <NavSection title="Machines" required={[FLAGS.CLUSTER_API, FLAGS.MACHINE_CONFIG, FLAGS.CAN_LIST_MACHINE_CONFIG]}>
+        <NavSection title="Compute" required={[FLAGS.CLUSTER_API, FLAGS.MACHINE_CONFIG, FLAGS.CAN_LIST_MACHINE_CONFIG]}>
```

Comment:
We'll need to update the required logic here. This will hide the entire nav section if the machine API is not there, but we would still want to show nodes in that case.

Reply:
Pushed updates